core/src/main/java/io/kestra/core/docs/AbstractClassDocumentation.java

```diff
@@ -114,6 +127,24 @@ protected static Map<String, Object> flatten(Map<String, Object> map, List<Strin
         return result;
     }
 
+    // Some task can have the `type` property but not to represent the task
+    // so we cant to keep it in the doc
+    private Boolean isTypeToKeep(String key){
+        List<Class<?>> typesToKeep = List.of(
+            AbstractRetry.class
+        );
```

Comment:
Well, I just ask to be extracted to a method, not need to create a list with one element this seems overcomplicated ;)

Reply:
Well, if we face another issue similar with a type property, we just need to add `Class.class` and its done

Reply:
If we face another issue we just nee to switch to a list and iterate over it.
KISS, don't optimize for a potential future that will never happen